src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts

```diff
@@ -180,6 +180,16 @@ export const walkthroughs: GettingStartedWalkthroughContent = [
 		content: {
 			type: 'steps',
 			steps: [
+				{
+					id: 'settingsSync',
+					title: localize('gettingStarted.settingsSync.title', "Sign in to get started with VS Code"),
```

Comment:
May be we should say here **Personalise your VS Code** or **Make VS Code your own** instead?

Reply:
"Personalise your VS Code" sounds good to me.